content/projects/git-exercises/_index.md

```diff
@@ -5,6 +5,7 @@ prerequisites:
   hard: ["topics/git"]
   soft: []
 tags: ["git"]
+story_points: 1
```

Comment:
Normally git is not as straight forward to recruits so. It does require quite a bit of effort from them to fully understand what is actually happening and why we use git. I'd suggest story point 2 here.

Reply:
We are not rating the learning curve but rather the effort of the assignment. Also this and linux is the basis off which all others are rated